cmd/kubeadm/app/cmd/phases/init/uploadcerts.go

```diff
@@ -36,7 +36,6 @@ func NewUploadCertsPhase() workflow.Phase {
 		Run:   runUploadCerts,
 		InheritFlags: []string{
 			options.CfgPath,
-			options.NodeCRISocket,
```

Comment:
Revert #114443, as `NodeCRISocket` is not used by `upload-certs` phase.

Reply:
Should we clean the release note of https://github.com/kubernetes/kubernetes/pull/114443 as you revert it？

Reply:
Thanks, yes, I have set the release note to `NONE`